gulpfile.js

```diff
@@ -135,6 +135,7 @@ declareExtension('amp-soundcloud', '0.1', {hasCss: false});
 declareExtension('amp-springboard-player', '0.1', {hasCss: false});
 declareExtension('amp-sticky-ad', '1.0', {hasCss: true});
 declareExtension('amp-story', '0.1', {hasCss: true});
+declareExtension('amp-story-auto-ads', '0.1', {hasCss: false});
```

Comment:
I think we agreed not to have this as a separate binary. Can you move the code into the amp-story extension?

Reply:
sure, I guess we can move it later if it bloats